### DISCUSSION TITLE
Could someone explain errors and how to handle them please?

### DISCUSSION THREAD

**Discussion by JordanChoe** [marcusolsson/obsidian-projects#412] [Help]
Edited to correct the DND file additions. 

Hello,  one of my projects for books is showing errors. I searched for errors in the forum and nothing populated. 

I have a few templates, one for when Booksearch works, one for when it doesn't (Booksearch plugin issue that doesn't escape question marks in titles), and one I have simply been fiddling with called book status notes before finding Booksearch plugin.  I'm having to scroll right in project inspector to see where the column issue is, which might be a screen resolution issue on my end vs the project inspector not being wide enough.  

My apologies for having to screenshot the scroll right.  Selecting the text in the error box isn't possible.

However, I cannot figure out how to correct these errors.  I tried deleting the yaml field kyledone from two of the templates and it simply popped a new error for kalidone. These are to be true/false yaml fields. The project inspector isn't really helping me here, including the one asking for a uniquevalue that already is unique to the yaml fields in these files.

I'm attaching screenshots so hopefully someone can help me determine how to correct these so those notes will display?  I'm also attaching the related md files to the errors, if that helps, including the two notes that are not templates throwing errors. Excluding the templates from the project somehow might fix those errors, but it wouldn't affect the other two, correct?

Would attaching my project settings be helpful?

Thank you.

![Screenshot (98)](https://user-images.githubusercontent.com/108248499/214907282-c8cbfeb2-2ef8-4b8d-a7ad-4e6f17bda8d5.png)
![Screenshot (99)](https://user-images.githubusercontent.com/108248499/214907284-b4aaca6d-4d71-4e40-96ae-cae0b4fcc49c.png)
![Screenshot (100)](https://user-images.githubusercontent.com/108248499/214907287-4c78f991-5a4c-4e23-ad37-d85003fe3fc8.png)
![Screenshot (101)](https://user-images.githubusercontent.com/108248499/214907290-de7a4d17-e665-48f0-bdd9-6101f16c479f.png)

[Mini Farming Self-Sufficiency on quarter acre- Brett L Markham.md](https://github.com/marcusolsson/obsidian-projects/files/10511925/Mini.Farming.Self-Sufficiency.on.quarter.acre-.Brett.L.Markham.md)

[Book Template no booksearch.md](https://github.com/marcusolsson/obsidian-projects/files/10511930/Book.Template.no.booksearch.md)
[books status note.md](https://github.com/marcusolsson/obsidian-projects/files/10511931/books.status.note.md)
[Logic of English Phonograms Game Book.md](https://github.com/marcusolsson/obsidian-projects/files/10511932/Logic.of.English.Phonograms.Game.Book.md)
[Book Template.md](https://github.com/marcusolsson/obsidian-projects/files/10511933/Book.Template.md)

**Comment by marcusolsson** [comment]
Hm. From the error, I would've guessed that you have duplicate keys in the frontmatter, but looking at the note, it doesn't look like it. I'll need to do some troubleshooting on this.
